ArticleTemplates/assets/js/modules/youtube.js

```diff
@@ -0,0 +1,246 @@
+                touchPoint.addEventListener('click', playVideo.bind(null, id, players[id].placeholder.parentNode));
+            } else {
+                touchPoint.addEventListener('click', sendPlayEventForNativePlayer.bind(null, id));
+            }
+            
+            players[id].placeholder.classList.add('fade-touchpoint');
+        } else {
+            players[id].iframe.parentNode.classList.add('show-video');
+        }
+    }
+
+    function sendPlayEventForNativePlayer(id) {
+        trackEvent({
+            id: id,
+            eventType: 'video:content:start'
+        });
+    }
+
+    function playVideo(id, placeholderParent) {
+        players[id].player.playVideo();
+        placeholderParent.classList.add('show-video');
+        setTimeout(hidePlaceholder.bind(null, placeholderParent), 300);
+    }
+
+    function hidePlaceholder(placeholderParent) {
+        placeholderParent.classList.add('hide-placeholder');
+    }
+
+    function onPlayerStateChange(id, event) {
+        Object.keys(stateHandlers).forEach(checkState.bind(null, id, event.data));
+    }
+
+    function checkState(id, state, stateHandler) {
+        if (state === window.YT.PlayerState[stateHandler]) {
+            stateHandlers[stateHandler](id);
+        }
+    }
+
+    function onPlayerPlaying(id) {
+        var currentTime = Math.round(players[id].player.getCurrentTime());
+
+        stopPlayers(id);
+        setProgressTracker(id);
+
+        if (currentTime === 0) {
+            trackEvent({
+                id: id,
+                eventType: 'video:content:start'
+            });
+        }
+    }
+
+    function onPlayerEnded(id) {
+        var placeholderParent; 
+
+        if (players[id].placeholder) {
+            placeholderParent = players[id].placeholder.parentNode;
+            placeholderParent.classList.remove('hide-placeholder');
+            setTimeout(showPlaceholder.bind(null, placeholderParent), 1000);
+        }
+
+        killProgressTracker(false, id);
+
+        trackEvent({
+            id: id,
+            eventType: 'video:content:end'
+        });
+    }
+
+    function showPlaceholder(placeholderParent) {
+        placeholderParent.classList.remove('show-video');
+    }
+
+    function onPlayerPaused(id) {
+        killProgressTracker(false, id);
+    }
+
+    function stopPlayers(ignoreId) {
+        Object.keys(players).forEach(stopPlayer.bind(null, ignoreId));
+    }
+
+    function stopPlayer(ignoreId, stopId) {
+        if (ignoreId !== stopId) {
+            players[stopId].player.pauseVideo();
+        }
+    }
+
+    function recordPlayerProgress(id) {
+        var currentTime = players[id].player.getCurrentTime(),
+            percentPlayed = Math.round(((currentTime / players[id].duration) * 100));
+
+        if (percentPlayed > 0 &&
+            percentPlayed % 25 === 0 &&
+            players[id].trackingCalls.indexOf(percentPlayed) === -1) {
```

Comment:
It appear `100` event is not supported by Ophan, so you shouldn't fire it:
https://github.com/guardian/ophan/blob/1d70ed327d13a0b5dbea4dd0d28f4d0db3fea68f/the-slab/app/extractors/MediaPlaybackExtractor.scala#L38-L41